Language/Structure/Control Structure/switchCase.adoc

```diff
@@ -71,9 +76,11 @@ switch (var) {
   case 1:
     //do something when var equals 1
     break;
-  case 2:
```

Comment:
Since the "Notes and Warnings" section comes after the "Example Code" section, I think it would be better to leave the example code as it was and instead add a code snippet in the "Notes and Warnings" section to demonstrate what you mean. This could just be the case.

Reply:
Yes, I think I got the changes in the right place now?